vowpalwabbit/Makefile

```diff
@@ -1,4 +1,4 @@
-BINARIES = vw active_interactor
+BINARIES = vw vwpredict active_interactor
```

Comment:
Namewise, maybe vw_slim_predict?  We want to be clear that this does not replace vw -t in general.

Organizationally, maybe we should have the specialized parts in a subdirectory?

Reply:
that far it's only vw_predict.* files. should I move to ../vw_slim_predict folder?

Reply:
Let's do that, along with the make details.